overlord/configstate/configcore/experimental.go

```diff
@@ -48,8 +48,13 @@ func validateExperimentalSettings(tr config.Conf) error {
 	return nil
 }
 
-func ExportExperimentalFlags(tr config.Conf) error {
-	dir := dirs.FeaturesDir
+func ExportExperimentalFlags(tr config.ConfGetter, opts *ApplyOptions) error {
```

Comment:
this forces us somewhat to make ApplyOptions public. We should probably instead  have this one as:

doExportExperimentalFlags

and have ExportExperimentalFlags(tr config.Conf) call doExperimentalFlags, same for Run and Apply

Reply:
Done